### PR TITLE
Pytorch 1.8 update bokeh version

### PR DESCRIPTION
**Pull request by Qingzi-Lan**
### Description
Bokeh 2.4.2 updates its dependency 'jquery-ui' to v1.13.0 to include security fixes

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu [hunk 1]
```diff
@@ -129,7 +129,7 @@ ENV PATH="$OPEN_MPI_PATH/bin:$PATH"
 ENV LD_LIBRARY_PATH=$OPEN_MPI_PATH/lib/:$EFA_PATH/lib/:$LD_LIBRARY_PATH
 
 RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value \
- && curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && curl -L -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py37_4.10.3-Linux-x86_64.sh \
```
Comment (arjkesh) [inline pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu:132]: Can you add comments here describing the version pin and also why there is "py37" in this specific sh script?

File: pytorch/inference/docker/1.8/py3/cu111/Dockerfile.gpu [hunk 2]
```diff
@@ -20,7 +20,6 @@ ENV LANG C.UTF-8
 ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
 ENV PATH /opt/conda/bin:$PATH
 ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
-ENV TEMP=/home/model-server/tmp
```
Comment (SergTogul) [inline pytorch/inference/docker/1.8/py3/cu111/Dockerfile.gpu:23]: Does it break anything or just not needed?

File: pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu (hunk 1 above)
Comment (SergTogul) [reply pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu:133]: Could you also add a few words into the PR description. Why are you switching to anaconda repo and why do you pin the script version

File: pytorch/inference/docker/1.8/py3/cu111/Dockerfile.gpu (hunk 2 above)
Comment (Qingzi-Lan) [reply pytorch/inference/docker/1.8/py3/cu111/Dockerfile.gpu:23]: the TEMP variable is not used in the dockerfile. removed as it's not needed

File: pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu (hunk 1 above)
Comment (Qingzi-Lan) [reply pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu:133]: added in dockerfile, will add it in description also

**Review by SergTogul — DISMISSED** [review]

**Review by SergTogul — APPROVED** [review]

**Review by arjkesh — APPROVED** [review]
